plugins/plugin-cpp/che-plugin-cpp-lang-ide/src/main/java/org/eclipse/che/plugin/cpp/ide/CppLocalizationConstant.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.cpp.ide;
+
+import com.google.gwt.i18n.client.Messages;
+
+/**
+ * Localization constants. Interface to represent the constants defined in resource bundle:
+ * 'CppLocalizationConstant.properties'.
+ *
+ * @author Vitalii PArfonov
+ */
+public interface CppLocalizationConstant extends Messages {
+
+
+    @Key("cpp.action.create.c.file.title")
```

Comment:
You can remove `@Key` annotation since you've used `@DefaultMessage`
It's not used and redundant